### DISCUSSION TITLE
questions on loading network policies

### DISCUSSION THREAD

**Discussion by cwrx777** [k3s-io/k3s#10300] [Q&A]
[Reference](https://docs.k3s.io/security/hardening-guide#networkpolicies)
> Network policies should be placed the /var/lib/rancher/k3s/server/manifests directory, where they will automatically be deployed on startup.

1. Can network policy be applied after k3s starts, i.e using `kubectl apply -f`'?
2. If it must be provided during startup, can the network policy file be located in a custom folder and the path is passed to k3s as command line argument or config file?

**Comment by brandond** [comment]
You can add manifests to that directory at any time. See https://docs.rke2.io/helm#automatically-deploying-manifests-and-helm-charts
